[New] Function Wiki (#200)
NilPointer-Software/bdfd-wiki: src/bdscript/publishMessage.md

# $publishMessage
Publishes a message from an announcement channel to all following servers.

## Syntax
```
$publishMessage[Channel ID;Message ID]
```
- `Channel ID` : The ID of the channel where the message is.

> - `Message ID` : The ID of the message which needs to be publish.

Reviewer: ```suggestion
- `Message ID` : The ID of the message which will be published.
```
